src/test/java/cloudBankTestCases/ClientCreationTestCase.java

```diff
@@ -0,0 +1,99 @@
+package cloudBankTestCases;
+
+import cloudBankPages.ClientListPage;
+import cloudBankPages.CreateClientPage;
+import cloudBankPages.Homepage;
+import cloudBankPages.LogInPage;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.time.Duration;
+
+import static org.testng.Assert.assertTrue;
+
+public class ClientCreationTestCase {
+
+    WebDriver driver;
+
+    LogInPage objLogInPage;
+    Homepage objHomePage;

```

Comment:
Do these need to be initialized here? Can they be done locally instead?

Reply:
I retained this part as I needed it to be global. Please refer to the latest changes I made where I consolidated all related functionalities in one class.

ex. In ClientTestCase class, instead of creating instances of Homepage class for each method, I  declared a global object "**Homepage homepage**" and instantiated it inside the setup() method.